Resync web-platform-tests/dom/observable

keithamus opened the pull request:
#### 117bd2d95f0392743ef11674f749332c2485262f
<pre>
Resync web-platform-tests/dom/observable
<a href="https://bugs.webkit.org/show_bug.cgi?id=276708">https://bugs.webkit.org/show_bug.cgi?id=276708</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/c4ba02f2ea432bfa1036b2923d75dffe11a06fe2">https://github.com/web-platform-tests/wpt/commit/c4ba02f2ea432bfa1036b2923d75dffe11a06fe2</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-constructor.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-constructor.any.js:
(test.t.source.subscribe.complete):
(test.t.source.subscribe.error):
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-constructor.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any.js: Added.
(promise_test.async const):
(promise_test):
(promise_test.async let):
(promise_test.async t):
(createTestSubject):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any.js: Added.
(promise_test.async let):
(promise_test):
(promise_test.async const):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any.js: Added.
(promise_test.async const):
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any.js: Added.
(test):
(createTestSubject):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.window.js: Added.
(async loadIframeAndReturnContentWindow):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any.js: Added.
(test):
(promise_test.async const):
(promise_test):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any.js: Added.
(promise_test.async const):
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.window.js: Added.
(async loadIframeAndReturnContentWindow):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any.js: Added.
(promise_test.async let):
(promise_test):
(promise_test.async const):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any.js: Added.
(test):
(createTestSubject):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-takeUntil.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-takeUntil.any.js:
(promise_test.async const):
(promise_test):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-takeUntil.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-toArray.any.js:
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/w3c-import.log:
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/dom/observable/observable-from.any-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any-expected.txt: Added.
* LayoutTests/platform/mac-wk2/imported/w3c/web-platform-tests/dom/observable/observable-from.any-expected.txt: Added.

Canonical link: <a href="https://commits.webkit.org/281344@main">https://commits.webkit.org/281344@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c7d7ab1e1b0367c4804450792a9cfa4ba7ec38d3

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/59595 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/38940 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/12119 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/63510 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/10118 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/46593 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/10271 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/48352 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/7082 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/61625 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/36348 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/51588 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/29190 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/33051 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/8840 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/9042 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/54998 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/9118 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/65242 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/3523 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/9005 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/55694 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/3534 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/51581 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/55820 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/13205 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/2923 "Passed tests") | | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/34754 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/35837 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/36923 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/35582 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Please make sure it's green before landing

keithamus committed:
Resync web-platform-tests/dom/observable
https://bugs.webkit.org/show_bug.cgi?id=276708

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/c4ba02f2ea432bfa1036b2923d75dffe11a06fe2

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-constructor.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-constructor.any.js:
(test.t.source.subscribe.complete):
(test.t.source.subscribe.error):
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-constructor.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-drop.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any.js: Added.
(promise_test.async const):
(promise_test):
(promise_test.async let):
(promise_test.async t):
(createTestSubject):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-every.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-filter.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any.js: Added.
(promise_test.async let):
(promise_test):
(promise_test.async const):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-find.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any.js: Added.
(promise_test.async const):
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-first.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any.js: Added.
(test):
(createTestSubject):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-flatMap.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-forEach.window.js: Added.
(async loadIframeAndReturnContentWindow):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any.js: Added.
(test):
(promise_test.async const):
(promise_test):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-inspect.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any.js: Added.
(promise_test.async const):
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-last.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-map.window.js: Added.
(async loadIframeAndReturnContentWindow):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any.js: Added.
(promise_test.async let):
(promise_test):
(promise_test.async const):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-some.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any.js: Added.
(test):
(createTestSubject):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-switchMap.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any.js: Added.
(test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-take.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-takeUntil.any-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-takeUntil.any.js:
(promise_test.async const):
(promise_test):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-takeUntil.any.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-toArray.any.js:
(promise_test):
* LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/w3c-import.log:
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/dom/observable/observable-from.any-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/observable/tentative/observable-from.any-expected.txt: Added.
* LayoutTests/platform/mac-wk2/imported/w3c/web-platform-tests/dom/observable/observable-from.any-expected.txt: Added.

Canonical link: https://commits.webkit.org/281344@main
